polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -831,6 +834,8 @@ struct LoopInternalization
   /// nullptr if the loop doesn't need to be versioned.
   Value getVersionCondition(LoopLikeOpInterface loop,
                             const WorkGroupSize &workGroupSize,
+                            std::variant<Value, unsigned> reqdSharedMemory,
+                            const unsigned sharedMemoryRemaining,
```

Comment:
I qualified the variable as `const` to prevent modification inside the member function given that this variable is logically immutable. Of course using `const` is moot from the caller perspective. But perhaps I'm missing some fine point... 

Note: I can change it to remove `const` but I am just curious if I'm missing something.